src/Sylius/Behat/Context/Ui/Admin/ManagingCustomersContext.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Context\Ui\Admin;
+
+use Behat\Behat\Context\Context;
+use Sylius\Behat\Page\Admin\Crud\UpdatePageInterface;
+use Sylius\Behat\Page\Admin\Crud\IndexPageInterface;
+use Sylius\Behat\Page\Admin\Customer\CreatePageInterface;
+use Sylius\Behat\Service\NotificationCheckerInterface;
+use Sylius\Component\User\Model\CustomerInterface;
+use Webmozart\Assert\Assert;
+
+/**
+ * @author Anna Walasek <anna.walasek@lakion.com>
+ */
+final class ManagingCustomersContext implements Context
+{
+    const RESOURCE_NAME = 'customer';
+
+    /**
+     * @var IndexPageInterface
+     */
+    private $indexPage;
+
+    /**
+     * @var CreatePageInterface
+     */
+    private $createPage;
+
+    /**
+     * @var NotificationCheckerInterface
+     */
+    private $notificationChecker;
+
+    /**
+     * @var UpdatePageInterface
+     */
+    private $updatePage;
+
+    /**
+     * @param CreatePageInterface $createPage
+     * @param IndexPageInterface $indexPage
+     * @param UpdatePageInterface $updatePage
+     * @param NotificationCheckerInterface $notificationChecker
+     */
+    public function __construct(
+        CreatePageInterface $createPage,
+        IndexPageInterface $indexPage,
+        UpdatePageInterface $updatePage,
+        NotificationCheckerInterface $notificationChecker
+    ) {
+        $this->createPage = $createPage;
+        $this->indexPage = $indexPage;
+        $this->updatePage = $updatePage;
+        $this->notificationChecker = $notificationChecker;
+    }
+
+    /**
+     * @Given I want to create a new customer
+     */
+    public function iWantToCreateANewCustomer()
+    {
+        $this->createPage->open();
+    }
+
+    /**
+     * @When /^I specify its ([^"]*) as "([^"]*)"$/
```

Comment:
:+1: but change ``[^"]*`` to ``[^"]+`` to ensure, that at least one word i typed ;)